hack/local-up-kubeedge.sh

```diff
@@ -255,7 +255,7 @@ if [[ "${ENABLE_DAEMON}" = false ]]; then
 else
     while true; do
         sleep 3
-        kubectl get nodes | grep edge-node | grep -q Ready && break
+        kubectl get nodes | grep edge-node | grep -q ^Ready$ && break
```

Comment:
```suggestion
        kubectl get nodes | grep edge-node | grep -q -w Ready && break
```
Just had a double check, `-w` can also do regexp match :)

Reply:
> Just had a double check, `-w` can also do regexp match :)

yes. `-w` is better here.